sonar-csharp-plugin/pom.xml

```diff
@@ -119,6 +119,12 @@
     <dependency>
       <groupId>org.sonarsource.sonarqube</groupId>
       <artifactId>sonar-plugin-api-impl</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
`sonar-plugin-api-impl` embeds junit 4 and this allowed our tests to run. We have to manually exclude it to ensure that we replaced all the usages.